libs/blocks/quiz/quiz.css

```diff
@@ -625,25 +625,135 @@ span.quiz-Button-label {
 /* Added to prevent click event triggering below the cards 
 TODO: Need to check if we can skip this somehow.
 */
-.consonant-CardsGrid--with4xGutter {
-  padding-bottom: 0px !important;
+.quiz-container .consonant-CardsGrid--with4xGutter {
+  padding-bottom: 0px;
+}
+
+.card .consonant-OneHalfCard-title {
```

Comment:
In most places, I see that you've scoped your changes to depend on the .quiz-container wrapper, but not everywhere. Since caas-uar.css loads after quiz.css, and just because we never know when someone will dump a card collection into a quiz page just to be perverse, it seems like we should be really methodical about using .quiz-container whenever we make updates to generic caas or consonant classes. 
<img width="970" alt="Screenshot 2023-06-06 at 2 27 08 PM" src="https://github.com/adobecom/milo/assets/25230829/f56ba0a4-4863-41d1-9014-567224e73cc6">